Libraries/CMSIS/Device/Maxim/MAX32520/Source/GCC/max32520.mk

```diff
@@ -86,3 +86,6 @@ LIBPATH+=$(CMSIS_ROOT)/Device/Maxim/$(TARGET_UC)/Source/GCC
 
 # Include the rules and goals for building
 include $(CMSIS_ROOT)/Device/Maxim/$(TARGET_UC)/Source/GCC/gcc.mk
+
+# Include rules for flashing
+include $(CMSIS_ROOT)/../../Tools/Flash/flash.mk
```

Comment:
I was going to yell at you for using `CMSIS_ROOT` instead of `MAXIM_PATH` but I see that the rest of the file is the real problem so you're off the hook.